src/js/components/disable-feature-button.js

```diff
@@ -0,0 +1,34 @@
+/* global ClassifAI */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ *  ClassifAI Disable Feature Link
+ *
+ * @param {Object} props         The block props.
+ * @param {string} props.feature The feature to disable.
+ */
+export const DisableFeatureButton = ( { feature } ) => {
+	// Check if user has permission to disable feature.
+	if (
+		! feature ||
+		! ClassifAI?.opt_out_enabled_features?.includes( feature )
+	) {
+		return null;
+	}
+
+	return (
+		<Button
+			href={ ClassifAI?.profile_url }
```

Comment:
Would be great to link down to the ClassifAI section, which I think should be as simple as adding `#classifai-profile-features-section` to the end of this URL (either here or where we localize this data)

Reply:
done.